res/menu/upload_files.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
Same in main_menu.xml.
Anyway already changed in upload_files.xml



Allan@dingyantec.com
 
From: Bartosz Przybylski
Date: 2015-12-18 10:06
To: owncloud/android
CC: Allan
Subject: Re: [android] Add select all file when upload file to remote (#1342)
In res/menu/upload_files.xml:
> @@ -0,0 +1,11 @@
> +<?xml version="1.0" encoding="utf-8"?>
> +<menu xmlns:android="http://schemas.android.com/apk/res/android"
> +      xmlns:app="http://schemas.android.com/apk/res-auto">

i mean app namespace isn't showAsAction part of android namespace ?
—
Reply to this email directly or view it on GitHub.